libagent/ssh/client.py

```diff
@@ -59,22 +68,31 @@ def parse_ssh_blob(data):
         i = io.BytesIO(data[6:])
         # https://github.com/openssh/openssh-portable/blob/master/PROTOCOL.sshsig
         res['sshsig'] = True
+        res['sshcertsign'] = False
         res['namespace'] = util.read_frame(i)
         res['reserved'] = util.read_frame(i)
         res['hashalg'] = util.read_frame(i)
         res['message'] = util.read_frame(i)
     else:
-        i = io.BytesIO(data)
         res['sshsig'] = False
-        res['nonce'] = util.read_frame(i)
-        i.read(1)  # SSH2_MSG_USERAUTH_REQUEST == 50 (from ssh2.h, line 108)
-        res['user'] = util.read_frame(i)
-        res['conn'] = util.read_frame(i)
-        res['auth'] = util.read_frame(i)
-        i.read(1)  # have_sig == 1 (from sshconnect2.c, line 1056)
-        res['key_type'] = util.read_frame(i)
-        public_key = util.read_frame(i)
-        res['public_key'] = formats.parse_pubkey(public_key)
+        _certificate = certificate.parse(data)
+        if _certificate['isCertificate']:
+            certificate.format(_certificate)
```

Comment:
There is no `format()` defined in `libagent.ssh.certificate` module.

It produces the following error:

```
Traceback (most recent call last):
  File "/XYZ/lib/python3.10/site-packages/libagent/server.py", line 103, in handle_connection
    reply = handler.handle(msg=msg)
  File "/XYZ/lib/python3.10/site-packages/libagent/ssh/protocol.py", line 106, in handle
    reply = method(buf=buf)
  File "/XYZ/lib/python3.10/site-packages/libagent/ssh/protocol.py", line 147, in sign_message
    signature = self.conn.sign(blob=blob, identity=key['identity'])
  File "/XYZ/lib/python3.10/site-packages/libagent/ssh/__init__.py", line 279, in sign
    return conn.sign_ssh_challenge(blob=blob, identity=identity)
  File "/XYZ/lib/python3.10/site-packages/libagent/ssh/client.py", line 35, in sign_ssh_challenge
    msg = parse_ssh_blob(blob)
  File "/XYZ/lib/python3.10/site-packages/libagent/ssh/client.py", line 80, in parse_ssh_blob
    certificate.format(_certificate)
AttributeError: module 'libagent.ssh.certificate' has no attribute 'format'
```